Assets/Editor/FindMissingScriptsRecursively.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public class FindMissingScriptsRecursively : EditorWindow 
```

Comment:
Also it only searches the gameobject you have selected and children of that gameobject.